notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>I'm personally very curious about which steps in the calc_rotation_rate function take the most time... can we optimize? I will investigate after finishing the initial review. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>I think that it's downloading the lightcurve. Thomas had mentioned something about the cloud speedup — I'll review my notes.</p>